paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -179,6 +180,68 @@ class PaymentSheet internal constructor(
         IntentConfirmationInterceptor.createIntentCallback = createIntentCallback
     }
 
+    /**
+     * Builder to add optional callbacks to [PaymentSheet].
+     *
+     * @param paymentResultCallback Called with the result of the payment after [PaymentSheet] is dismissed.
+     */
+    class Builder(internal val resultCallback: PaymentSheetResultCallback) {
```

Comment:
You'll need to regenerate the API files for these changes `./gradlew apiDump`.

Reply:
Done 👍